scheduler/test/cook/test/mesos/api.clj

```diff
@@ -538,7 +538,7 @@
       (is (<= 200 (:status initial-get-resp) 299)))
 
     (testing "update changes quota"
-      (let [new-quota {:cpus 9.0 :mem 4323.0 :count 43 :gpus 3.0}
+      (let [new-quota {:cpus 9.0 :mem 4323.0 :count 43.0 :gpus 3.0}
```

Comment:
Count is now a float instead of an int. This is kind of a contentious change I guess, but we were previously in a weird state where the default value was a double but the database field was an int, so if you were calling something like`JsonElement.getAsInt()` it would fail. I thought changing it to a double would be more consistent.

Reply:
We discussed offline, I converted this back to an int and changed the default to Integer/MAX_VALUE